pkg/rtc/participant.go

```diff
@@ -2328,3 +2329,25 @@ func (p *ParticipantImpl) ClearInProgressAndProcessSubscriptionRequestsQueue(tra
 
 	go p.ProcessSubscriptionRequestsQueue(trackID)
 }
+
+func (p *ParticipantImpl) UpdateSubscribedQuality(nodeID livekit.NodeID, trackID livekit.TrackID, maxQualities []types.SubscribedCodecQuality) error {
```

Comment:
These are moved to `LocalParticipant` as for relay track they do not go through `RemoteParticipant`, but directly to the relay manager.